### ISSUE TITLE
Run post-merge builds with same configuration

### ISSUE THREAD

**Issue by ChristianKuehnel** [google/llvm-premerge-checks#169]
To see what is failing on master, it would be useful to run post-merge tests with the same configuration as the pre-merge tests.

**Comment by joker-eph** [comment]
Any update on bots availability to run on master?

Keeping this configuration always green is important to reduce the number of false positive on the pre-merge testing and avoid developing a habit of ignoring the "red" Harbormaster (related to #193)

**Comment by ChristianKuehnel** [comment]
I suppose once we have the builds ported to Buildkite, it would be easy to trigger builds on master.

**Comment by joker-eph** [comment]
LG! Any ETA by the way?

**Comment by joker-eph** [comment]
Maybe setting up the master build could be a first good setup on BuildKite before the pre-merge?

**Comment by metaflow** [comment]
@joker-eph  https://buildkite.com/llvm-project/llvm-master-build <- here . It's actually in place from the beginning of June but somehow I forgot to tell you. The build cannot blame a specific commit and only show results. Also some configurations are disabled on windows as they are constantly failing.

**Comment by ChristianKuehnel** [comment]
Here's a proposal on what that could look like: 
https://github.com/google/llvm-premerge-checks/blob/main/docs/monitoring_main_branch.md

**Comment by metaflow** [comment]
Open questions (from doc)

1. Do we actually need to create a BuilderFactory or can we also just run a shell script?
2. How can we quickly test and iterate on the Python scripting? How can we run our own Buildbot master for testing?
3. The current solution by only wrapping the existing pre-merge testing scripts will not result in nice build steps as the other builders are producing as buildkite does not have that concept. Is that an issue?
4. We do need to check out the pre-merge testing config and scripts form another git repo. Is that an issue?

**Comment by metaflow** [comment]
5. is it possible to disable batching and run tests on every commit?

**Comment by joker-eph** [comment]
> is it possible to disable batching and run tests on every commit?

Yes, it is even the recommended way I think. See the best practices: https://llvm.org/docs/HowToAddABuilder.html#best-practices-for-configuring-a-fast-builder

> Enable batch builds
> As a last resort, you can configure your builder to batch build requests. This makes the build failure notifications markedly less actionable, and should only be done once all other reasonable measures have been taken.
